packages/universal-components/src/RangeDatePicker/RenderDay.js

```diff
@@ -76,17 +123,18 @@ export default function RenderDay({
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
+    flex: parseFloat(defaultTokens.zIndexDefault),
```

Comment:
I think we should return `1` here. `zIndexDefault` is not really related to `flex` (even if it's currently 1).

Reply:
Lol :D true

Reply:
Fixed